python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_withoutobservedinputs.py

```diff
@@ -0,0 +1,175 @@
+# This is why we create a sample with size equal to 5 and dimension equal to 0. 
+# Even if the calibration model has no inputs observations, observed inputs 
+# are required by the current programming interface and this is why we have to 
+# create this object. 
+inputObservations = ot.Sample(size, 0)  # Trick
+
+# %%
+# We are now ready to perform the calibration.
+algo = ot.LinearLeastSquaresCalibration(
+    model, inputObservations, outputObservations, candidate, "SVD"
+)
+algo.run()
+calibrationResult = algo.getResult()
+parameterMAP = calibrationResult.getParameterMAP()
+print(parameterMAP)
+
+
+# %%
+# We observe that the estimated parameter is relatively close to 
+# the true parameter value.
+print(parameterMAP - trueParameter)
+
+
+# %%
+# Graphical validation
+# --------------------
+#
+# We now validate the calculation by drawing the exact function and compare 
+# it to the function with estimated parameters.
+fullModel = ot.SymbolicFunction(["a", "b", "c", "x"], ["a + b * x + c * x^2"])
+parameterIndices = [0, 1, 2]
+trueFunction = ot.ParametricFunction(fullModel, parameterIndices, trueParameter)
+print(trueFunction)
+beforeCalibrationFunction = ot.ParametricFunction(fullModel, parameterIndices, candidate)
+print(beforeCalibrationFunction)
+calibratedFunction = ot.ParametricFunction(fullModel, parameterIndices, parameterMAP)
+print(calibratedFunction)
+
+# %%
+# In order to validate the calibration, we compare the parametric function 
+# with true parameters at given abscissas with the parametric function 
+# with calibrated parameters. 
+abscissas = [-1.0, -0.6, -0.2, 0.2, 0.6, 1.0]
+xmin = min(abscissas)
+xmax = max(abscissas)
+
+npoints = 50
+graph = ot.Graph("Calibration without observations", "x", "y", True, "bottomright")
+curve = trueFunction.draw(xmin, xmax, npoints).getDrawable(0)
+curve.setLineStyle("dashed")
+curve.setLegend("True model")
+curve.setColor("darkorange1")
+graph.add(curve)
+# Before calibration
+curve = beforeCalibrationFunction.draw(xmin, xmax, npoints)
+curve.setLegends(["Model before calibration"])
+curve.setColors(["red"])
+graph.add(curve)
+# After calibration
+curve = calibratedFunction.draw(xmin, xmax, npoints)
+curve.setLegends(["Model after calibration"])
+curve.setColors(["green"])
+graph.add(curve)
+# Observations
+for i in range(outputDimension):
+    cloud = ot.Cloud(ot.Sample([[abscissas[i]]] * size), outputObservations[:, i])
+    cloud.setColor("blue")
+    if i == 0:
+        cloud.setLegend("Observations")
+    graph.add(cloud)
+viewer = viewer.View(graph)
+
+plt.show()
+
+
+# %%
+# We notice that the calibration produces a good fit to the data. 
+# Still, we notice a small discrepancy between the true mode land the model 
+# after calibration, but this discrepancy is very small.
+# 
+# Actually, this was made much easier from the fact that the model is 
+# linear with respect to the calibrated parameters a, b and c.
```

Comment:
I am not sure this remark is appropriate. Also, a b and c should be written as math expressions to keep notations consistent. What is easier? And easier compared to what?

Reply:
I must add the following comment: "Since the model is linear with respect to the parameters a, b, c, therefore the LinearLeastSquares method performs well. If the modal was non linear w.r.t. a, b, c, therefore the linear least squares method would not work that well and the parameters would be estimated with less accuracy."

Reply:
Fixed.